itl/command-plugins.conf

```diff
@@ -2505,6 +2505,20 @@ object CheckCommand "ldap" {
 			set_if = "$ldap_verbose$"
 			description = "Show details for command-line debugging"
 		}
+		"-4" = {
+			set_if = "$ldap_ipv4$"
+			description = "Use IPv4 connection"
+		}
+		"-6" = {
+			set_if = "$ldap_ipv6$"
+			description = "Use IPv6 connection"
+		}
+		"-A" = {
+			# Need to come after -T or -S
+			order = 99
+			value = "$ldap_certexpiry$"
+			description = "warn,critical days before cert expiry"
+ 		}
```

Comment:
I can't find this parameter anywhere in the plugin [documentation](https://www.monitoring-plugins.org/doc/man/check_ldap.html).

Reply:
Both CentOS and Homebrew (on the mac) uses https://github.com/nagios-plugins/nagios-plugins and they have a '-A' option:
https://github.com/nagios-plugins/nagios-plugins/blob/master/plugins/check_ldap.c#L418

Hmmm can we document the difference in the description?

Reply:
Since this plugin is not documented in the official ``Icinga 2`` documentation, it is better that we add to the built in checkCommands parameters that are supported by the plugin documented [here](https://icinga.com/docs/icinga-2/latest/doc/10-icinga-template-library/) and if a user uses another plugin for this then you can configure a custom checkCommand with the appropriate parameters.

Reply:
I've got a better idea: https://github.com/Icinga/icinga2/commit/c4a436a244193e379eeb1ab4415cc5adb66c0b44

Reply:
I can understand Icinga 2 doc explicitly mention Monitoring plugins and knowing how touchy this subject is I'm not going to pursuit further - I have what I need in my environment and this PR can be used for future reference. Feel free to close this. 

Though @Al2Klimov is right that we _could_ doc better ;-)

Reply:
Why close? Just change as in https://github.com/Icinga/icinga2/commit/c4a436a244193e379eeb1ab4415cc5adb66c0b44 .

Reply:
Sorry @Al2Klimov I am not really familiar with icinga nor I fully understand what you mean. Are you suggesting that we update the doc and mark the -A option only works with nagios-plugins?

Reply:
Exactly.